.github/workflows/enso4igv.yml

```diff
@@ -128,6 +128,9 @@ jobs:
           java-version: "21"
           distribution: "graalvm-community"
 
+      - name: Clean sbt
```

Comment:
Ensure that generated sources are removed before running `sbt publishM2`.

Reply:
I am afraid `publishM2` will generate them again: https://github.com/enso-org/enso/actions/runs/12634477624/job/35202416216?pr=11770#step:9:5565 and it has to generate all the code otherwise we would be able to use such generated classes in `runtime-parser.jar`, right?

If there is a way to disable ScalaDoc (or replace it with Javadoc`) or restrict it somehow, then go that route. We certainly don't need `runtime-parser-javadoc.jar` for anything useful.